src/rcore.c

```diff
@@ -2952,7 +2952,7 @@ Vector2 GetWorldToScreenEx(Vector3 position, Camera camera, int width, int heigh
     }
     else if (camera.projection == CAMERA_ORTHOGRAPHIC)
     {
-        float aspect = (float)CORE.Window.screen.width/(float)CORE.Window.screen.height;
+        float aspect = ((double)width/(double)height);
```

Comment:
Should be `double aspect = `.

Reply:
don't width and height need to be floats instead?

Reply:
If you're asking about the base `CORE.Window.screen` dimensions, they are `Size` ([L393](https://github.com/raysan5/raylib/blob/master/src/rcore.c#L393)). `Size` is `unsigned int` ([L358](https://github.com/raysan5/raylib/blob/master/src/rcore.c#L358)). AFAIK, this is correct. I'm unaware of window sizes with float point dimensions (e.g.: [reference](https://www.glfw.org/docs/3.3/window_guide.html#window_size)).

If you're asking about [R2955](https://github.com/raysan5/raylib/pull/3351/files#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R2955) specifically, ahead it will multiply `aspect` ([R2957](https://github.com/raysan5/raylib/pull/3351/files#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R2957)) by a `double` ([R2956](https://github.com/raysan5/raylib/pull/3351/files#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R2956)). So either keep as:
```
double aspect = ((double)width/(double)height);
```
Or keep as floats (and let [R2957](https://github.com/raysan5/raylib/pull/3351/files#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R2957) promote the result to `double`):
```
float aspect = ((float)width/(float)height);
```